frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -187,19 +187,20 @@ end
 
 -- Returns document metadata (opened document or book (file) metadata or custom metadata).
 function BookInfo.getDocProps(ui, file, book_props, no_open_document, no_customize)
-    local original_props, filepath, display_title
+    local original_props, filepath
     if ui then -- currently opened document
         original_props = ui.doc_settings:readSetting("doc_props")
         filepath = ui.document.file
-        -- if original title is empty, generate it as filename without extension
-        display_title = original_props.title or filemanagerutil.splitFileNameType(filepath)
     else -- from file
         original_props = BookInfo.getBookProps(file, book_props, no_open_document)
         filepath = file
     end
 
     local props = no_customize and original_props or BookInfo.customizeProps(original_props, filepath)
-    props.display_title = display_title
+    if ui then
+        -- if original title is empty, generate it as filename without extension
+        props.display_title = props.title or filemanagerutil.splitFileNameType(filepath)
+    end
     return props
```

Comment:
> not called on the currently opened document

That means we call Book information from the file browser on long-pressing on a file.
We agreed that Book information shows original title.

Reply:
Still agreeing on that :)
Being not familiar (I already forgot what I reviewed a few days ago :/) with how this is used, I just wondered if there were other cases where this would be called. And you tell me there is none, so ok.
(But even if we don't display `display_title` in "Book information", may be it should be there for consistency and by contract? I wrote after my review:
> BookInfo:getDocProps(ui, file) : previous one, overriden with customized stuff, including display_title, and available as ui.doc_props

)

Reply:
EDIT: there is one place when we do need `display_title` from the file browser - screensaver showing book title in the message. To be fixed.